cli/src/main/java/com/devonfw/tools/ide/commandlet/VersionGetCommandlet.java

```diff
@@ -55,36 +55,45 @@ public void run() {
 
     ToolCommandlet commandlet = this.tool.getValue();
     VersionIdentifier configuredVersion = commandlet.getConfiguredVersion();
+    VersionIdentifier installedVersion = commandlet.getInstalledVersion();
```

Comment:
It seems that VersionGetCommandlet and EditionGetCommandlet are 95% identical.
We should move the code to a shared abstract base class `AbstractVersionOrEditionGetCommandlet`.